mwdb/resources/metakey.py

```diff
@@ -394,6 +398,82 @@ def put(self, key):
         schema = MetakeyDefinitionManageItemResponseSchema()
         return schema.dump(metakey_definition)
 
+    @requires_authorization
+    @requires_capabilities(Capabilities.manage_users)
+    def put(self, key):
+        """
+        ---
+        summary: Updated attribute key
+        description: |
+            Updated attribute key definition.
+
+            Requires `managing_attributes` capability.
+        security:
+            - bearerAuth: []
+        tags:
+            - attribute
+        parameters:
+            - in: path
+              name: key
+              schema:
+                type: string
+              description: Attribute key
+        requestBody:
+            description: Attribute key definition
+            content:
+              application/json:
+                schema: MetakeyDefinitionItemRequestBodySchema
+        responses:
+            200:
+                description: When metakey definition is successfully updated
+                content:
+                  application/json:
+                    schema: MetakeyDefinitionManageItemResponseSchema
+            400:
+                description: |
+                    When one of attribute definition fields is missing or incorrect.
+            403:
+                description: When user doesn't have `managing_attributes` capability.
+            404:
+                description: When metakey doesn't exist.
+        """
+        schema = MetakeyUpdateRequestSchema()
+        obj = loads_schema(request.get_data(as_text=True), schema)
+
+        metakey_obj = load_schema({"key": key}, MetakeyKeySchema())
+        metakey = (
+            db.session.query(MetakeyDefinition)
+            .filter(MetakeyDefinition.key == metakey_obj["key"])
+            .first()
+        )
+        if metakey is None:
+            raise NotFound("No such metakey")
+
+        label = obj["label"]
+        if label is not None:
+            metakey.label = label
+
+        description = obj["description"]
+        if description is not None:
+            metakey.description = description
+
+        url_template = obj["template"]
+        if url_template is not None:
+            metakey.url_template = url_template
+
+        hidden = obj["hidden"]
+        if hidden is not None:
+            if hidden == "Enabled":
```

Comment:
Hmm, why not fields.Boolean? Passing such strings is quite odd...
```suggestion
            if hidden == "Enabled":
```